src/main/java/cms/gov/madie/measure/utils/QiCoreJsonUtil.java

```diff
@@ -316,6 +316,8 @@ public static String updateResourceFullUrls(TestCase testCase, String madieJsonR
     return testCase.getJson();
   }
 
+  // update full urls for non-patient resources
+
```

Comment:
is this intentional? the comment does not match the description of the method below it if that's what the comment is for.

Reply:
it was not!